corehq/apps/reports_core/filters.py

```diff
@@ -197,7 +197,7 @@ def __init__(self, name, required=True, label='Choice List Filter',
         self.choices = choices or []
 
     def value(self, **kwargs):
-        choice = kwargs[self.name]
+        choice = str(kwargs[self.name])
```

Comment:
What about unicode? Not sure if the value would ever contain non-ascii characters.